### ISSUE TITLE
Mark good and bad should be exclusive in bisect

### ISSUE THREAD

**Issue by espr14** [gitextensions/gitextensions#10195]
### Feature description

User can mark specific commit as good and as bad. This brings warning in command line but it does not make sense. There are few options:
1. Add `reset bisect mark` which removes all bisect marks at the commit.
2. If the commit is marked as bad and user marks it as good then remove the bad mark.

### Environment

- Git Extensions 3.5.4.12724
- Build 65f01f39982ccb121c4df7d4b00b506939553958
- Git 2.35.1.windows.2
- Microsoft Windows NT 10.0.19044.0
- .NET Framework 4.8.4515.0
- DPI 96dpi (no scaling)

**Comment by mstv** [comment]
Could you describe the steps how to reproduce the problem and the actual and the expected behavior, please?

**Comment by espr14** [comment]
1. start bisect
2. pick a commit
3. mark it as good
4. mark the same commit as bad
5. an error will appear in the command line

![bisect1](https://user-images.githubusercontent.com/46957311/190980734-9ff4603e-70b6-4306-8595-92ab6039f246.png)

Expected behavior: After marking it as bad, `git extensions` would remove the `good` mark. No error should appear.

`bad`, `good`, and `skip` should be exclusive so it should not be possible to have more than 1 mark per commit.

**Comment by vbjay** [comment]
I agree this is a problem but I feel it is more a git issue. Git should
reset other flags if setting other flags. I'd raise it up there too.
